app/index.md

```diff
@@ -1,56 +1,55 @@
 ---
 layout: home
-title: Pilot service guidance
+title: Service guidance
 analytics_key: home
 ---
 
-During the pilot phase, we’ll use your feedback to test and improve the NHS Record a vaccination service (RAVS).
+The Record a vaccination service (RAVS) allows you to record COVID-19, Flu, Pertussis and RSV patient and staff vaccinations.  
 
-The service will be available at the same time as other point of care (POC) systems. But you only need to record data in RAVS.
-
-The service only records data. You have control over clinical judgements or decisions
+The service only records data. You have control over clinical judgements or decisions. 
 
 {% from 'action-link/macro.njk' import actionLink %}
 
 {{ actionLink({
-  "text": "Log in to the pilot service",
+  "text": "Log in to the pilot service (at www.ravs.england.nhs.uk)",
   "href": "https://www.ravs.england.nhs.uk"
 }) }}
 
-The pilot service is at www.ravs.england.nhs.uk
-
 {{ actionLink({
   "text": "How to use the pilot service",
   "href": ("/guide/" | url)
 }) }}
 
-## Access training
+{{ actionLink({
+  "text": "How to onboard organisations",
+  "href": ("/onboarding-organisations/" | url)
+}) }}
```

Comment:
Suggest we remove this for now as that page is added in the other branch, and then add it back in as part of the other branch.

```suggestion
```